.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -192,6 +192,24 @@ steps:
 
         rm -f new_test_plan_id.txt
 
+        # all feature branches branched out from master can use py3only image
+        parent_commit=$(git rev-parse HEAD^1)
+        master_commit=$(git rev-parse master)
+        branch_from_master=0
+        if [ "$parent_commit" == "$master_commit" ]
```

Comment:
Would that be better to compare with the branch but not commit? Like branch above 202405
Once a branch like 202411 branched out from master in the future, and certainly it could use py3only image, but compare commit id would cause branch_from_master=0 and use internal image

Reply:
That's right. Thanks. I will probably use a list of known branches that use mixed and the rest can use py3only.